selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java

```diff
@@ -237,18 +181,58 @@ private Locators() {}
     }
   }
 
-  @FindBy(id = Locators.EDITOR_TABS_PANEL)
-  private WebElement editorTabsPanel;
+  /** Editor`s markers types */
+  public enum MarkersType {
+    ERROR_MARKER_OVERVIEW("//div[@class='ruler annotations']/div[@class='annotation error']"),
+    WARNING_MARKER_OVERVIEW("//div[@class='ruler annotations']/div[@class='annotation warning']"),
+    TASK_MARKER_OVERVIEW("//div[@class='ruler annotations']/div[@class='annotation task']"),
+    ERROR_MARKER("//div[@class='ruler overview']/div[@class='annotationOverview error']"),
+    WARNING_MARKER("//div[@class='ruler overview']/div[@class='annotationOverview warning']"),
+    INFO_MARKER("//div[@class='annotationHTML info']");
+
+    private final String itemLocator;
+
+    MarkersType(String itemLocator) {
+      this.itemLocator = itemLocator;
+    }
+
+    private String get() {
+      return this.itemLocator;
+    }
+  }
+
+  /** Editor's context menu items */
+  public enum EditorContextMenu {
```

Comment:
`Editor` could be omitted in _enum_ name so as it's part of `CodenvyEditor` class.